src/startupscriptgenerator/dotnetcore/scriptgenerator.go

```diff
@@ -88,8 +88,10 @@ func (gen *DotnetCoreStartupScriptGenerator) GenerateEntrypointScript() string {
 			return ""
 		}
 	}
-
-	return scriptBuilder.String()
+	var runScript = scriptBuilder.String()
+	logger.LogInformation("Now logging generated startup script.")
```

Comment:
Consolidate these two messages to one, to align with `BuildCommand.cs`:
`logger.LogDebug("Build script content:\n" + scriptContent);`